pegasus/sites.v3/hourofcode.com/views/promote_handouts.haml

```diff
@@ -5,15 +5,17 @@
   - hoc_one_pager = localized_file('/files/hoc-one-pager.pdf')
   - schools_handout = localized_file('/files/schools-handout.pdf')
 
+%h2.handouts-header= hoc_s(:handouts_section)
+
 %a{:id => "onepager"}
 %div.handout.col-50
-  %h2= hoc_s(:handouts_onepager)
+  %div.handout-text= hoc_s(:handouts_onepager)
```

Comment:
%div is actually implied if you just do .handout-text which is a bit more readable.